### PR TITLE
ROX-13165: Removing security overview section and redesigning network security section

### PR DESCRIPTION
**Pull request by sachaudh**
## Description

Making modifications in the DeploymentDetails component based on the updated mocks shown here: https://docs.google.com/presentation/d/1DdKZ0PoLk6yDyV9vFM4Q-HpB0N_lXwJ53G-CYAc56Sc/edit#slide=id.g1669bc43c7d_13_51

## Checklist
- [x] Investigated and inspected CI test results
- [ ] ~Unit test and regression tests added~
- [ ] ~Evaluated and added CHANGELOG entry if required~
- [ ] ~Determined and documented upgrade steps~
- [ ] ~ Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs)) ~

## Screenshots

Screen Shot 2022-10-18 at 1 26 18 PM
Screen Shot 2022-10-18 at 1 26 15 PM

### CODE REVIEW

File: ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentDetails.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ import {
     TextContent,
     TextVariants,
 } from '@patternfly/react-core';
-import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import InfoCircleIcon from '@patternfly/react-icons/dist/esm/icons/info-circle-icon';
```
Comment (vjwilson) [inline ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentDetails.tsx:20]: Is this linking to the single icon file (lodash-style) the recommended way to do it? 
(Mainly want to make sure PF team guarantees that the path should never change.)

**Review by vjwilson — APPROVED** [review]
Just a little question about the icon import

Keep calm and carry on

File: ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentDetails.tsx (hunk 1 above)
Comment (sachaudh) [reply ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentDetails.tsx:20]: Yea basically. Seems like the PF examples do something similar too. That's a good question. Maybe I'll keep the same style used in the line above for now

**Pull request merged**